splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -105,81 +101,102 @@ SplunkRum initialize(
         OpenTelemetryRumBuilder otelRumBuilder = OpenTelemetryRum.builder();
 
         otelRumBuilder.setResource(createResource());
-        initializationEvents.add(
-                new RumInitializer.InitializationEvent(
-                        "resourceInitialized", startupTimer.clockNow()));
+        initializationEvents.emit("resourceInitialized");
 
         CurrentNetworkProvider currentNetworkProvider =
                 currentNetworkProviderFactory.apply(application);
-        initializationEvents.add(
-                new InitializationEvent("connectionUtilInitialized", startupTimer.clockNow()));
+        initializationEvents.emit("connectionUtilInitialized");
+
+        // TODO: How truly important is the order of these span processors? The location of event
+        // generation should probably not be altered...
```

Comment:
Hah, good question. Ideally we should have some sort of a contract that dictates what we can do with these (or, no contract meaning we can do whatever)